src/app/globals.css

```
@@ -3509,3 +3509,111 @@ html.custom-cursor-active * {
+   --edge-l / --edge-r are written from the scroll handler: 1 is a fully opaque
+   edge (nothing hidden that way), 0 a fully faded one. Defaulting both to 1
+   means the mask is a no-op until the first scroll, so a row that has only
+   just overflowed never flashes a fade it hasn't earned.
+
+   The scrollbar is hidden because the fades, the wheel-to-travel conversion and
+   native touch/keyboard scrolling all already expose the affordance, and a
+   permanent gutter under the tabs (Windows, Linux) would reintroduce exactly
+   the boxed-in look this row is meant to avoid. */
+.category-strip-scroller {
+  --edge-w: 2.5rem;
+  scrollbar-width: none;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)